git/BUILD.bazel

```diff
@@ -1 +1,8 @@
-package(default_visibility = ["//visibility:public"])
\ No newline at end of file
+package(default_visibility = ["//visibility:public"])
+
+filegroup(
```

Comment:
probably don't need this addition

Reply:
The `BUILD` file containing a `filegroup` is required as part of the testing framework.

Reply:
On 2nd though I'll create the `filegroup` within the testing framework dynamically.
Removed the `BUILD` file.

Reply:
The `BUILD` file is required on the `rules/git` folder when `git_cached_repository` is loaded before usage. 
Reverting change and adjusting tests accordingly.

```
ERROR: error loading package '': 
Every .bzl file must have a corresponding package, but '@wix_build_tools//rules/git:git_cached_repository.bzl' does not have one.
```